utils/installer/install.sh

```diff
@@ -61,17 +91,25 @@ EOF
 
   __add_separator "80"
 
-  echo "Would you like to install lunarvim's NodeJS dependencies?"
-  read -p "[y]es or [n]o (default: no) : " -r answer
-  [ "$answer" != "${answer#[Yy]}" ] && install_nodejs_deps
+  if [ -z "$ARGS_INSTALL_NONINTERACTIVE" ]; then
+    echo "Would you like to install lunarvim's NodeJS dependencies?"
+    read -p "[y]es or [n]o (default: no) : " -r answer
+    [ "$answer" != "${answer#[Yy]}" ] && install_nodejs_deps
 
-  echo "Would you like to install lunarvim's Python dependencies?"
-  read -p "[y]es or [n]o (default: no) : " -r answer
-  [ "$answer" != "${answer#[Yy]}" ] && install_python_deps
+    echo "Would you like to install lunarvim's Python dependencies?"
+    read -p "[y]es or [n]o (default: no) : " -r answer
+    [ "$answer" != "${answer#[Yy]}" ] && install_python_deps
 
-  echo "Would you like to install lunarvim's Rust dependencies?"
-  read -p "[y]es or [n]o (default: no) : " -r answer
-  [ "$answer" != "${answer#[Yy]}" ] && install_rust_deps
+    echo "Would you like to install lunarvim's Rust dependencies?"
+    read -p "[y]es or [n]o (default: no) : " -r answer
+    [ "$answer" != "${answer#[Yy]}" ] && install_rust_deps
+  else
+    if [ "$ARGS_INSTALL_NONINTERACTIVE" == y ]; then
```

Comment:
There's no need to validate this again since it can only take a pre-set value. You can declare as read-only if you want.

Reply:
fixed in 2d28c78